macros/utils/graph/set_cache_entry.sql

```diff
@@ -0,0 +1,6 @@
+{% macro set_cache_entry(entry, val) %}
```

Comment:
Basically everything that we store on the graph is caching so maybe let's store everything inside `graph["elementary"]`? Not crucial though.